startupscript/butane/001-install-node.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# install-node.sh installs Node.js on the VM.
```

Comment:
I think it would be nice for anyone who will maintain this to have some indication motivation for the installation mechanism. Otherwise, they'll look at this and potentially compare it to official "how to install Node on Linux" documentation and spend some time re-discovering what you already ran into.

Reply:
done